src/Transit.js

```diff
@@ -0,0 +1,13 @@
+import BusStop from './BusStop.js'
+
+
+function Transit() {
+  
+    return(<div>
+      <h1>TRANSIT</h1>
+      <BusStop stopName="Hennepin & 24th" stopId="1328" />

```

Comment:
UI question: will the 2 bus stops be distinguishable to the user?

Reply:
updated names to be more clear